src/components/Button/Button.tsx

```diff
@@ -83,7 +98,7 @@ const ButtonWrapper = styled.TouchableOpacity(
 )
 
 const Label = styled.Text(({theme, color}: {color?: string; theme?: ITheme}) => ({
-  color: color || 'white',
+  color: color,
```

Comment:
same with borderRadius
```suggestion
  color,
```

Reply:
Updated